sample_size/scripts/sample_size_run.py

```diff
@@ -15,19 +15,32 @@ def main() -> None:
     from sample_size.sample_size_calculator import SampleSizeCalculator
     from sample_size.scripts.input_utils import get_alpha
     from sample_size.scripts.input_utils import get_metric_metadata
+    from sample_size.scripts.input_utils import get_mde
+    from sample_size.scripts.input_utils import get_variants
+    from sample_size.scripts.input_utils import register_another_metric
 
     try:
         # Get alpha for power analysis
         alpha = get_alpha()
-        calculator = SampleSizeCalculator(alpha)
+        variants = get_variants()
```

Comment:
can we check if we need to update readme? https://github.com/godaddy/sample-size#script-constraints
(also I suggest having two separate PRs for easier review)